_studio/mfx_lib/shared/src/mfx_common_decode_int.cpp

```diff
@@ -191,8 +142,16 @@ void ConvertMFXParamsToUMC(mfxVideoParam const* par, UMC::VideoStreamInfo *umcVi
     if (par->mfx.FrameInfo.PicStruct & MFX_PICSTRUCT_FIELD_TFF)
         umcVideoParams->interlace_type = UMC::INTERLEAVED_TOP_FIELD_FIRST;
 
+    umcVideoParams->stream_type = UMC::UNDEF_VIDEO;
+    switch (par->mfx.CodecId)
+    {
+        case MFX_CODEC_AVC:   umcVideoParams->stream_type = UMC::H264_VIDEO;  break;
+        case MFX_CODEC_HEVC:  umcVideoParams->stream_type = UMC::HEVC_VIDEO;  break;
+        case MFX_CODEC_MPEG2: umcVideoParams->stream_type = UMC::MPEG2_VIDEO; break;
+        case MFX_CODEC_VC1:   umcVideoParams->stream_type = UMC::VC1_VIDEO;   break;
+        case MFX_CODEC_VP9:   umcVideoParams->stream_type = UMC::VP9_VIDEO;   break;
```

Comment:
I would add default branch to avoid compiler warnings and static analysis cries:
```
switch (par->mfx.CodecId)
{
....
default: umcVideoParams->stream_type = UMC::UNDEF_VIDEO; break;
}

```

Reply:
Compilers are good with current code. But, given https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/Coding_guidelines.md#common-mistakes I'll fix it.